config/config/sc-config.yaml

```diff
@@ -640,6 +640,7 @@ opensearch:
     min: 7
     max: 14
     ageSeconds: 864000
+    maxRequestSeconds: 600
```

Comment:
If we get timeout when we've used this. Should we increase it? If so I think we also need to change the activeDeadline for this as well.

Reply:
Yes we'll need to change the deadlineseconds as well :)

Reply:
Yes true about the activeDeadline. I did not change the default value as it works for some environments. But i can just double it.

Reply:
I still think that we need to increase the activeDeadline. Because right now it's hardcoded to 600 seconds so it will kick in before the request timeout.

Reply:
It is was already set to 45min https://github.com/elastisys/compliantkubernetes-apps/blob/main/config/config/sc-config.yaml#L646

Reply:
Right, I missed that! But I still think that the default values in the chart should match https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/charts/opensearch/slm/values.yaml#L15